src/styles/main.scss

```diff
@@ -1,3 +1,180 @@
+* {
+  box-sizing: border-box;
+}
+
 body {
   margin: 0;
+  font-family: "Roboto", serif;
+  font-weight: 400;
+  line-height: 18px;
+}
+
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap");
+
+p {
+  margin: 0;
+}
+
+.header {
+  height: 60px;
+  background: #fff;
```

Comment:
Move all styles related to `header` to separate file.
One block -> one styles file.